Compiler/AST/EqualsBlock.cs

```diff
@@ -0,0 +1,54 @@
+﻿using CompiledHandlebars.Compiler.AST.Expressions;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using CompiledHandlebars.Compiler.Visitors;
+
+namespace CompiledHandlebars.Compiler.AST
+{
+    internal class EqualsBlock : BlockWithElse
```

Comment:
Add a class `EqualsExpression` here.
It should contain two expressions which can later be compared: `lhs` and `rhs`